config/webhook/manifests.yaml

```diff
@@ -23,3 +23,20 @@ webhooks:
     - CREATE
     resources:
     - pods
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /mutate-pvcs
+  failurePolicy: Ignore
```

Comment:
I think we still need to decide if this should be ignore or fail. Mostly likely fail because we don't want to create a misconfigured volume.

Reply:
Yes, good point.  We don't want to silently create a volume without encryption if it was requested.  Changed.

Reply:
Setting the default to fail has the side-effect that the e2e tests for the other controllers fail unless the webhook is started.  I'd rather not have the e2e tests depend on other controllers so setting the default back to ignore.

Longer-term we should create a new envtest for each controller with just the config that it requires.  This means moving envtest setup from `BeforeSuite` to `BeforeEach`.